Earthfile

```diff
@@ -1,11 +1,12 @@
+VERSION 0.6
 
 INSTALL_DIND:
     COMMAND
     COPY +install-dind-script/install-dind.sh /tmp/install-dind.sh
     RUN /tmp/install-dind.sh
 
 install-dind-script:
-    FROM alpine:3.13
+    FROM alpine:latest
```

Comment:
Having it float could be a little bit of an issue for reproducibility (i realize the 3.13 is still a mutable tag, but its at least somewhat pinned.